pkg/libsmbios.spec.in

```diff
@@ -299,17 +299,15 @@ ln -s %{_sbindir}/dellWirelessCtl %{buildroot}/%{_bindir}/dellWirelessCtl
 ln -s smbios-sys-info %{buildroot}/%{_sbindir}/getSystemId
 ln -s smbios-wireless-ctl %{buildroot}/%{_sbindir}/dellWirelessCtl
 ln -s smbios-lcd-brightness %{buildroot}/%{_sbindir}/dellLcdBrightness
-ln -s smbios-rbu-bios-update %{buildroot}/%{_sbindir}/dellBiosUpdate
 
 cat > files-python-smbios <<-EOF
-	%doc COPYING-GPL COPYING-OSL README
+	%doc COPYING-GPL COPYING-OSL
 	%{python_sitelib}/*
 EOF
 
 cat > files-smbios-utils-python <<-EOF
-	%doc COPYING-GPL COPYING-OSL README
+	%doc COPYING-GPL COPYING-OSL
 	%doc src/bin/getopts_LICENSE.txt src/include/smbios/config/boost_LICENSE_1_0_txt
-	%doc doc/pkgheader.sh
```

Comment:
I can't find any hint of this actually existing git grepping through the entire history of the codebase returned nothing.